compiler/model/metamodel.ts

```diff
@@ -133,7 +133,7 @@ export class Property {
    * If specified takes precedence over `name` when generating code. `name` is always the value
    * to be sent over the wire
    */
-  identifier?: string
+  codegen_name?: string
```

Comment:
This should be `codegenName` (we use camelCase in the metamodel), here and in 2 other places in this file.